python/libkvikio/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# =============================================================================
+# Copyright (c) 2022-2024, NVIDIA CORPORATION.
```

Comment:
Please double-check the other files in this PR as well, I'm sure there are other new files.
```suggestion
# Copyright (c) 2024, NVIDIA CORPORATION.
```